docs/source/examples/10/inputs.py

```diff
@@ -0,0 +1,48 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from pathlib import Path
+
+import numpy as np
+from sklearn import datasets
+
+from health_azure import get_workspace
+from health_azure.datasets import get_datastore
+
+
+def main() -> None:
+    # import some data to play with
+    iris = datasets.load_iris()
+    X = iris.data[:, :2]  # we only take the first two features.
+    y = iris.target
+    inputs = Path("inputs")
+    inputs.mkdir(exist_ok=False)
+    X_csv = inputs / "X.csv"
+    np.savetxt(X_csv, X, delimiter=',')
```

Comment:
maybe kill this whole script? Writing a dataset to file is already covered in another example